data/transactions/verify/streamverifier.go

```diff
@@ -295,127 +195,40 @@ func (sv *StreamVerifier) tryAddVerificationTaskToThePool(ue []*UnverifiedElemen
 	if l, c := sv.verificationPool.BufferSize(); l == c {
 		return false, nil
 	}
-	err = sv.addVerificationTaskToThePoolNow(ue)
+	err = sv.addVerificationTaskToThePoolNow(uElmts)
 	if err != nil {
 		// An error is returned when the context of the pool expires
 		return false, err
 	}
 	return true, nil
 }
 
-func (sv *StreamVerifier) addVerificationTaskToThePoolNow(ue []*UnverifiedElement) error {
+func (sv *StreamVerifier) addVerificationTaskToThePoolNow(unvrifiedElts []UnverifiedElement) error {
```

Comment:
I agree with this, and I merged your proposal.
I also renamed variables/functions to make this independent StreamToBatch operation.
As for moving it to util/execpool, it will be in a later phase (https://app.zenhub.com/workspaces/algorand-main-workspace-5ea71fda6d554d53dda36f16/issues/gh/algorand/go-algorand-internal/2778), because it will require more package level tests to have proper coverage of StreamToBatch, independent of txn sig verification.